### ISSUE TITLE
Feature request: allow selection of tasks > 100

### ISSUE THREAD

**Issue by govvin** [osmlab/maproulette3#1127]
![image](https://user-images.githubusercontent.com/5821688/75139884-c2feac80-5728-11ea-8a13-3810ad27595a.png)

When managing tasks, there doesn't seem to be a way to select matching tasks beyond 100 records.  I mistakenly thought that selecting "disabled" from the selection menu will select every matching disabled record beyond what's visible on the map:

![image](https://user-images.githubusercontent.com/5821688/75140038-140ea080-5729-11ea-9735-d9e7a79b76b6.png)

Currently, to go about selecting these records, I would have to repeat the process 100 records at a time, which could easily become tedious when working with thousands of records.

Or am I doing this wrong?

**Comment by nrotstan** [comment]
You're right, the checkbox in the table header only applies to the page of results you're viewing, which is counter-intuitive. I agree that if I landed on a challenge and selected all Too Hard Tasks and reset them to Created status, then I would expect that to apply to all Too Hard tasks in the challenge.

You also mention the map, though, and the map is another filter. When you first land on a challenge all tasks are shown on the map. If I zoomed in to a geographic region and then selected Too Hard tasks, I would expect only Too Hard tasks in that area to be selected (but still spanning multiple pages of task results if there were a lot).

I assume you're trying to change their status?
